python_modules/dagster/dagster/_core/definitions/assets.py

```diff
@@ -85,6 +85,10 @@
 ASSET_SUBSET_INPUT_PREFIX = "__subset_input__"
 
 
+def stringify_asset_key_to_input_name(asset_key: AssetKey) -> str:
```

Comment:
is there not an existing utility that does this? We should use a single, canonical one